msgid ""
msgstr ""
"Project-Id-Version: red-discordbot\n"
"POT-Creation-Date: 2021-05-25 15:06+0000\n"
"Last-Translator: \n"
"Language-Team: Arabic\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: redgettext 3.3\n"
"Plural-Forms: nplurals=6; plural=(n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5);\n"
"X-Crowdin-Project: red-discordbot\n"
"X-Crowdin-Project-ID: 289505\n"
"X-Crowdin-Language: ar\n"
"X-Crowdin-File-ID: 670\n"
"Language: ar_SA\n"

#: redbot/cogs/audio/core/events/cog.py:215
msgid "Auto Play started."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:40
msgid "You have attempted to run Audio's Lavalink server on an unsupported architecture. Only settings related commands will be available."
msgstr "لقد حاولت تشغيل خادم Lavalink للصوت على هندسة معمارية غير مدعومة. لن تتوفر سوى الأوامر ذات الصلة بالاعدادات."

#: redbot/cogs/audio/core/events/dpy.py:70
msgid "I'm missing permissions in this server, Please address this as soon as possible.\n\n"
"Expected Permissions:\n"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Enabled"
msgstr "تمكين"

#: redbot/cogs/audio/core/events/dpy.py:77
msgid "Disabled"
msgstr "معطل"

#: redbot/cogs/audio/core/events/dpy.py:124
msgid "No DJ role found. Disabling DJ mode."
msgstr "لم يتم العثور على دور DJ. تعطيل وضع DJ."

#: redbot/cogs/audio/core/events/dpy.py:134
msgid "`{user_input}` is not a valid value for `{command}`"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:142
msgid "Unable To Parse Argument"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:154
#: redbot/cogs/audio/core/events/dpy.py:163
msgid "Invalid Argument"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:155
msgid "The argument you gave for `{}` is not valid: I was expecting a `{}`."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:175
msgid "Invalid Environment"
msgstr "بيئة غير صالحة"

#: redbot/cogs/audio/core/events/dpy.py:176
msgid "Connection to Lavalink has been lost."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:184
msgid "No Player Available"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:185
msgid "The bot is not connected to a voice channel."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:193
#: redbot/cogs/audio/core/events/lavalink.py:162
msgid "Unable to Get Track"
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:194
#: redbot/cogs/audio/core/events/lavalink.py:163
msgid "I'm unable to get a track from Lavalink at the moment, try again in a few minutes."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:205
msgid "There was an issue communicating with Discord."
msgstr ""

#: redbot/cogs/audio/core/events/dpy.py:206
msgid "This error has been reported to the bot owner."
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:154
msgid "Couldn't get a valid track."
msgstr "تعذر الحصول على مسار صالح."

#: redbot/cogs/audio/core/events/lavalink.py:189
msgid "Now Playing"
msgstr "تشغيل الآن"

#: redbot/cogs/audio/core/events/lavalink.py:191
msgid "Track length: {length} | Requested by: {user}"
msgstr "طول المسار: {length} <unk> طلب من: {user}"

#: redbot/cogs/audio/core/events/lavalink.py:212
msgid "Queue ended."
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:264
msgid "Multiple Errors Detected"
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:265
msgid "Closing the audio player due to multiple errors being detected. If this persists, please inform the bot owner as the Audio cog may be temporally unavailable."
msgstr "إغلاق مشغل الصوت بسبب العديد من الأخطاء التي يتم الكشف عنها. إذا استمر هذا، يرجى إبلاغ مالك البوت بأن رمز الصوت قد يكون غير متوفر مؤقتاً."

#: redbot/cogs/audio/core/events/lavalink.py:279
msgid "Track Stuck"
msgstr "تعثر المسارات"

#: redbot/cogs/audio/core/events/lavalink.py:280
msgid "Playback of the song has stopped due to an unexpected error.\n"
"{error}"
msgstr ""

#: redbot/cogs/audio/core/events/lavalink.py:286
msgid "Track Error"
msgstr "خطأ في المسار"

